hack/verify-structured-logging.sh

```diff
@@ -48,15 +48,48 @@ while IFS='' read -r line; do
   migrated_packages+=("$KUBE_ROOT/$line")
 done < <(cat "${migrated_packages_file}")
 
-
+echo "Running structured logging static check on migrated packages"
 ret=0
 GOOS=linux    logcheck "${migrated_packages[@]}" || ret=$?
 GOOS=windows  logcheck "${migrated_packages[@]}" || ret=$?
 
 if [ $ret -eq 0 ]; then
-  echo "Structured logging static check is passed :)."
+  echo "Structured logging static check passed on migrated packages :)"
 else
   echo "Please fix above failures. You can locally test via:"
   echo "hack/verify-structured-logging.sh"
 fi
+
+# Ignore migrated packages as they are already tested
```

Comment:
this doesn't seem right..? we should be *enforcing* the rules on migrated packages, but non-migrated packages should be expected to fail?

Reply:
There are 2 parts:
First part enforces rules on migrated packages (migrated packages must not use unstructured logging and must use correct structured logging patterns). This prevents regressions in migrated packages.

Second part enforces rules on all other packages (non migrated packages can use unstructured logging or structured logging). This checks if all non migrated packages that are using structured logging are doing it correctly. It's useful for validating PRs done for migration.